### DISCUSSION TITLE
flutter里用pagview，怎么设置动画播放到多少帧暂停啊？或者说播放到什么进度暂停

### DISCUSSION THREAD

**Discussion by shengdongwang** [Tencent/libpag#1611] [Q&A] [answered]
flutter里用pagview，怎么设置动画播放到多少帧暂停啊？或者说播放到什么进度暂停

**Comment by kevingpqi123** [comment, marked as answer]
有两种实现方式：
1、pag 支持区间播放，具体可以看下 pag 的官方 demo 工程，上面有范例，播放特定的区间即可；
2、PAGView 或 PAGImageView 的播放有回调方法，可以获取到每一帧的渲染回调，在回调信息中获取进度以及暂停播放
